lib/api/website.js

```diff
@@ -34,8 +34,10 @@ function _errorActions(err, errorDocument, routingRules,
         objectKey, err.code);
     if (errRoutingRule) {
         // route will redirect
-        monitoring.promMetrics(
-            'GET', bucketName, err.code, 'getObject');
+        if (request.method === 'GET') {
```

Comment:
some monitoring is present in GET but not in HEAD. I'm not sure if it's expected to be like that or if I should put the same monitoring in HEAD.

Reply:
I think the same monitoring should be present for HEAD and GET, it's likely an overlook